code/modules/recycling/sortingmachinery.dm

```diff
@@ -98,11 +98,40 @@
 /obj/item/delivery/update_overlays()
 	. = ..()
 	if(sort_tag)
-		. += "[base_icon_state]_sort"
+		var/mutable_appearance/destination_tag = mutable_appearance(icon, "tag_[sort_tag]", layer = layer + 0.05)
+		switch(icon_state)
+			if("deliverypackage1")
+				destination_tag.pixel_x = 5
+				destination_tag.pixel_y = -4
```

Comment:
My understanding of this is, for items that are adjusting screenspace position as opposed to map position, we want to use `pixel_w` instead of `pixel_x`, and `pixel_z` instead of `pixel_y`. It shouldn't have a visible effect, but helps futureproof that against layering issues down the line, but test-compile just to be sure.

You'll want to tweak that for the other switch cases as well.

Reply:
Sure, I'll switch em all and test.